exporter/signalfxexporter/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signalfxexporter
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for SignalFx exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// AccessToken is the authentication token provided by SignalFx.
+	AccessToken string `mapstructure:"access_token"`
+
+	// Realm is the SignalFx realm where data is going to be sent to. The
+	// default value is "us0"
+	Realm string `mapstructure:"realm"`
+
+	// URL is destination to where SignalFx metrics will be sent to, it is
+	// intended for tests and debugging. It cannot be specified at the same time
+	// with Realm.
+	URL string `mapstructure:"url"`
```

Comment:
This should clarify whether the URL is the "base url" or the full url including the `/v2/datapoint` path.  If this thing ever supports events (I think a prerequisite of replacing the gateway), then if this is the full path there will need to be a separate config option for events.

Reply:
Per your comment it seems better to make this the "base url" and automatically append the appropriate path. I will do that soon.